src/views/homepage/index.tsx

```diff
@@ -45,122 +38,55 @@ const HomePageMobileMenu = () => {
+          url={merchandising.vault.url}
+          cardTitle={merchandising.vault.cardTitle}
+          description={merchandising.vault.description}
+          ctaText={merchandising.vault.ctaText}
+        />
+        <HashiConfGlobalSlot
+          description={merchandising.hashiconfGlobal.description}
+        />
       </MerchandisingSlots>
-
       <LearnSection
-        media={<InlineSvg src={badge} />}
-        heading="Become HashiCorp Vault Certified"
-        description={
-          <>
-            <Text size={300}>
-              As a Cloud Engineer specializing in security, development, or
-              operations, you can take the Vault Associate exam to validate your
-              knowledge of the basic concepts, skills, and use cases associated
-              with open source HashiCorp Vault. Or take the Vault Operations
-              Professional exam to demonstrate your proficiency with deploying,
-              configuring, managing, and monitoring HashiCorp Vault in
-              production.
-            </Text>
-            <Text size={300}>
-              Upon passing either exam, you can easily communicate your
-              proficiency and employers can quickly verify your results.
-            </Text>
-          </>
-        }
-        collectionCards={collectionCards}
+        imageSrc={learnSection.imageSrc}
+        heading={learnSection.heading}
+        description={learnSection.description}
+        collectionCards={learnSection.collectionCards}
+        link={learnSection.link}
       />
-
       <PreFooter
-        heading="Looking for help?"
-        description="We offer paid support, a free forum, and other community resources."
-        actions={[
-          {
-            icon: (
-              <IconSupport24 color="var(--token-color-foreground-highlight)" />
-            ),
-            heading: 'Support',
-            description: 'Open a support ticket',
-            link: 'https://support.hashicorp.com/hc/en-us',
-          },
-          {
-            icon: <IconHelp24 color="var(--token-color-packer-brand)" />,
-            heading: 'Forum',
-            description: 'Find your answer on the forum',
-            link: 'https://discuss.hashicorp.com/',
-          },
-          {
-            icon: <IconUser24 color="var(--token-color-nomad-brand)" />,
-            heading: 'Community',
-            description: 'Join our community',
-            link: 'https://www.hashicorp.com/community',
-          },
-        ]}
+        heading={preFooter.heading}
+        description={preFooter.description}
+        actions={preFooter.actions}
       />
     </div>
   )
 }
 
-function HomePageView({ collectionCards }: HomePageViewProps): ReactElement {
+function HomePageView({ content }: HomePageProps): ReactElement {
   return (
     <div className={s.homepage}>
       <HomePageMobileMenu />
-      <HomePageContent collectionCards={collectionCards} />
+      <HomePageContent {...content} />
```

Comment:
This is one of the few cases where props spreading felt fine to me; thanks to TypeScript, and since the `HomePageContent` component is in the same file.